# dwa.msg
bool replan_bool
bool planning
geometry_msgs/PoseStamped[] poses
